src/loadgenerator/requirements.txt

```diff
@@ -2,22 +2,31 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements.txt requirements.in
+#    pip-compile --output-file=requirements.txt requirements.in
 #
-certifi==2020.6.20       # via requests
+certifi==2020.6.20        # via geventhttpclient, requests
 chardet==3.0.4            # via requests
-click==7.1.2                # via flask
-flask==1.1.2              # via locustio
-gevent==20.6.2             # via locustio
+click==7.1.2              # via flask
+configargparse==1.2.3     # via locust
+flask-basicauth==0.2.0    # via locust
+flask==1.1.2              # via flask-basicauth, locust
+gevent==20.9.0            # via geventhttpclient, locust
+geventhttpclient==1.4.4   # via locust
 greenlet==0.4.17          # via gevent
-idna==2.10                 # via requests
+idna==2.10                # via requests
 itsdangerous==1.1.0       # via flask
 jinja2==2.11.2            # via flask
-locustio==0.999
+locust==1.2.3             # via -r requirements.in
 markupsafe==1.1.1         # via jinja2
-msgpack-python==0.5.6     # via locustio
-pyzmq==17.1.3             # via locustio
-requests==2.24.0          # via locustio
-six==1.15.0               # via locustio
-urllib3==1.25.10           # via requests
-werkzeug==0.16.1          # via flask
+msgpack==1.0.0            # via locust
+psutil==5.7.2             # via locust
+pyzmq==19.0.2             # via locust
+requests==2.24.0          # via locust
+six==1.15.0               # via geventhttpclient
+urllib3==1.25.10          # via requests
+werkzeug==1.0.1           # via flask, locust
+zope.event==4.5.0         # via gevent
+zope.interface==5.1.0     # via gevent
+
+# The following packages are considered to be unsafe in a requirements file:
+# setuptools
```

Comment:
should we just remove these 2 lines?

Reply:
This is an auto-generated file from `pip-compile`, so I'd prefer to leave it as-is. I'm not sure I completely understand the context of the lines though